pkgs/top-level/python-packages.nix

```diff
@@ -1938,6 +1938,8 @@ in {
 
   croniter = callPackage ../development/python-modules/croniter { };
 
+  crossplane = callPackage ../development/python-modules/crossplane { };
```

Comment:
Do you expect this to be used as a python module? If it is just a script we don't need and want this entry.

Reply:
It provides couple functions to parse nginx config.
https://pypi.org/project/crossplane/#crossplane-parse

Reply:
I can expect somebody building static analyzer on top of this.